django/contrib/postgres/aggregates/mixins.py

```diff
@@ -20,14 +20,22 @@ def resolve_expression(self, *args, **kwargs):
         return super().resolve_expression(*args, **kwargs)
 
     def as_sql(self, compiler, connection):
+        additional_sql_params = []
+
         if self.ordering:
-            self.extra['ordering'] = 'ORDER BY ' + ', '.join((
-                ordering_element.as_sql(compiler, connection)[0]
-                for ordering_element in self.ordering
-            ))
+            ord_clauses = []
+            for ordering_element in self.ordering:
+                ord_sql, ord_sql_params = ordering_element.as_sql(compiler, connection)
+                ord_clauses.append(ord_sql)
+                additional_sql_params.extend(ord_sql_params)
+
+            self.extra['ordering'] = 'ORDER BY ' + ', '.join(ord_clauses)
         else:
             self.extra['ordering'] = ''
-        return super().as_sql(compiler, connection)
+
```

Comment:
Please chop all unnecessary blank lines.

Reply:
Too much consended imho, but probably more in line with django code style.